addons/salvage2/salvage2.lua

```diff
@@ -30,142 +30,146 @@ require('tables')
 require('strings')
 require('maths')
 require('logger')
+texts = require 'texts'
```

Comment:
And if you wanna do me another favor, add some parens around this :)

Reply:
I just copy/pasted it from THTracker. I've updated it in both, and I'll do another push for THTracker later.